src/math/warpExposure.cc

```diff
@@ -657,7 +948,7 @@ int afwMath::warpCenteredImage(
     t += dt;
     std::cout <<srcImage.getWidth()<<"x"<<srcImage.getHeight()<<": "<< dt <<" "<< t <<std::endl;
 #else
-    int n = warpImage(destImage, srcImageCopy, affXYTransform, control, padValue);
+    int n = warpImage(destImage, srcImageCopy, affXYTransform, control, covImage, padValue);
```

Comment:
I don't understand why you've renamed some of these to `warpCovariance` and not others.  Do all of these variants compute both the image and the covariance now?